### ISSUE TITLE
PCmover Express is an invalid solution

### ISSUE THREAD

**Issue by Vectrexer** [MicrosoftDocs/windows-itpro-docs#583]
PCmover Express is an invalid solution.  The utility is no longer offered for free by Laplink.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: cde8580a-bd82-ebd2-3767-f3ee662a904e
* Version Independent ID: 94323219-e4ed-8761-723e-06be33c3b9cc
* Content: [User State Migration Tool (USMT) Overview (Windows 10)](https://docs.microsoft.com/en-us/windows/deployment/usmt/usmt-overview)
* Content Source: [windows/deployment/usmt/usmt-overview.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/deployment/usmt/usmt-overview.md)
* Service: **unspecified**
* GitHub Login: @greg-lindsay
* Microsoft Alias: **greglin**

**Comment by GMelford** [comment]
Yes. They did that in a short time to market their products. We have given up on Laplink as they do not handle the issues we have, even as a paying customer. USMTGUI use USMT and allow us to use our own XMLs.
With their latest version they also supply a range of customized XMLs AND they found a way to migrate WIN10 App settings and data.

And no their products are not free but who works for free? They are up to date, always help us solving our issues and constantly develop the SW. 
I have used them for nearly 6 years now.
